logos/src/logos/main.py

```
@@ -338,91 +495,178 @@ async def generalstats(data: LogosKeyModel):
+                result.model_id,
+                -1, # Policy ID not tracked yet
+                result.classification_stats,
+                result.scheduling_stats
+            )
+    except Exception as e:
+        _pipeline.record_completion(
+            request_id=result.scheduling_stats.get("request_id"),
+            result_status="ERROR",
+            error_message=str(e)
+        )
+        raise e
```

Ruff (0.14.7)

Use `raise` without specifying exception name

Remove exception name

(TRY201)